.gitignore

```diff
@@ -0,0 +1,5 @@
+*.pyc
+.coverage
+*,cover
```

Comment:
Should this be: *.cover

Reply:
The [pytest-cov](https://pypi.org/project/pytest-cov/) module creates a file named **file.py,cover** when you request code annotations `coverage annotate vectra.py.` That being said, I just looked and the functionality is being deprecated in favor of HTML reports which go in the **htmlcov** directory.

I'll remove this line and add the **htmlcov** line